src/components/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,151 @@
+  /**
+   * The date the countdown should end
+   */
+  endDate: string;
+  /**
+   * Any descriptor to appear below the coundown
+   */
+  intervalDescription?: string;
+  /**
+   * Text translation for days
+   */
+  daysLabel?: string;
+  /**
+   * Text translation for hours
+   */
+  hoursLabel?: string;
+  /**
+   * Text translation for minutes
+   */
+  minutesLabel?: string;
+  /**
+   * Text translation for seconds
+   */
+  secondsLabel?: string;
+  /**
+   * Text translation for what the countdown is for
+   */
+  label?: string;
+}
+/**
+ * ## Overview
+ *
+ * Overview of this widget
+ *
+ * [Figma Link](https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=10850-35410&m=dev)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/components-countdown--overview)
+ */
+const Countdown = forwardRef<HTMLDivElement, CountdownProps>(
+  (
+    {
+      endDate,
+      label = 'Lots Close in',
+      daysLabel = 'Days',
+      hoursLabel = 'Hours',
+      minutesLabel = 'Minutes',
+      secondsLabel = 'Seconds',
+      intervalDescription = 'Lots Close in 2-minute intervals',
+      className,
+      ...props
+    },
+    ref,
+  ) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps(props, 'Countdown');
+
+    const calculateTimeLeft = useCallback(() => {
+      const targetTime = new Date(endDate).getTime();
+      const currentTime = new Date().getTime();
+      const difference = targetTime - currentTime;
+
+      // Adjust for GMT offsets and Daylight Saving Time
+      const targetDate = new Date(endDate);
+      const currentDate = new Date();
+
+      const targetOffset = targetDate.getTimezoneOffset();
+      const currentOffset = currentDate.getTimezoneOffset();
+
+      const offsetDifference = (currentOffset - targetOffset) * 60 * 1000;
+      const adjustedDifference = difference + offsetDifference;
+
+      let timeLeft = {
+        days: 0,
+        hours: 0,
+        minutes: 0,
+        seconds: 0,
+      };
+
+      if (adjustedDifference > 0) {
+        timeLeft = {
+          days: Math.floor(adjustedDifference / (1000 * 60 * 60 * 24)),
+          hours: Math.floor((adjustedDifference / (1000 * 60 * 60)) % 24),
+          minutes: Math.floor((adjustedDifference / 1000 / 60) % 60),
+          seconds: Math.floor((adjustedDifference / 1000) % 60),
+        };
+      }
+
+      return timeLeft;
+    }, [endDate]);
+
+    const [timeLeft, setTimeLeft] = useState(calculateTimeLeft());
+
+    useEffect(() => {
+      const timer = setInterval(() => {
+        setTimeLeft(calculateTimeLeft());
+      }, 1000);
+
+      return () => clearInterval(timer);
+    }, [calculateTimeLeft, endDate]);
+
+    return (
+      <div {...commonProps} className={classnames(baseClassName, className)} {...props} ref={ref}>
+        <div className={`${baseClassName}__countdown-container`}>
```

Comment:
it looks like there is an aria-role for a timer: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/timer_role